br: public parameter --ignore-stats to backup statistic data (#15222)
pingcap/docs: br/br-snapshot-manual.md

    - [Restore a table](#restore-a-table)
    - [Restore multiple tables with table filter](#restore-multiple-tables-with-table-filter)
    - [Restore execution plan bindings from the `mysql` schema](#restore-execution-plan-bindings-from-the-mysql-schema)

> - [Restore statistics](#restore-statistics)

Reviewer: 中文里修改位置在
![image](https://github.com/pingcap/docs/assets/79440533/b1595ce7-45a5-4768-97b6-e0d528c39c7c)

英文需改在
![image](https://github.com/pingcap/docs/assets/79440533/cf839b3f-6713-4ae2-bad0-3c4863efa712)

因为英文翻译也需要是 Back up statistics

Author: fixed

ADMIN RELOAD BINDINGS;
```

> ## Restore statistics

Reviewer: 同上，位置和名字

Author: fixed

> Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. It supports backing up and restoring statistics of column, index, and table levels. After you enable this feature, you do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.

Author: ```suggestion
Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. When you set this parameter to `false', the `br` command-line tool supports backing up and restoring statistics of column, index, and table levels. You do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.
```

> Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. When you set this parameter to `false', the `br` command-line tool supports backing up and restoring statistics of column, index, and table levels. In this case, you do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.

Reviewer: ```suggestion
Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. When you set this parameter to `false', the `br` command-line tool supports backing up and restoring statistics of columns, indexes, and tables. In this case, you do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.
```

> This parameter is enabled by default, and the configuration item is `--ignore-stats=true`, which means that table statistics are not backed up when the data is backed up.

Author: ```suggestion
If you do not set this parameter to `false`, the `br` command-line tool uses the default setting `--ignore-stats=true`, which means statistics are not backed up during data backup.
```

The following is an example of backing up cluster snapshot data and backing up table statistics with `--ignore-stats=false`:

```shell
br backup full \
--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log \
--ignore-stats=false
```

> After completing the above configuration, when you restore data, if the table statistics are included in the backup set, the `br` command line tool automatically restores the table statistics during restoring data:

Author: ```suggestion
After completing the above configuration, when you restore data, if the table statistics are included in the backup, the `br` command line tool automatically restores the table statistics during restoring data:
```

> After completing the above configuration, when you restore data, if the table statistics are included in the backup, the `br` command-line tool automatically restores the table statistics during restoring data:

Reviewer: ```suggestion
After backing up data with the preceding configuration, when you restore data, the `br` command-line tool automatically restores table statistics if table statistics are included in the backup:
```

```shell
br restore full \
--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log
```

> When the backup and restore feature backs up data, it stores the statistics in JSON format in the `backupmeta` file. When restoring data, it imports the statistics in JSON format into the cluster. For more information, see [LOAD STATS](/sql-statements/sql-statement-load-stats.md).

Reviewer: ```suggestion
When the backup and restore feature backs up data, it stores statistics in JSON format within the `backupmeta` file. When restoring data, it loads statistics in JSON format into the cluster. For more information, see [LOAD STATS](/sql-statements/sql-statement-load-stats.md).
```
